yacc/console/yx_test.go

```diff
@@ -283,6 +283,16 @@ func TestRedistribute(t *testing.T) {
 			},
 			err: nil,
 		},
+		{
+			query: "REDISTRIBUTE KEY RANGE kr1 TO sh2 CHECK",
```

Comment:
`CHECK REDISTRIBUTE KEY RANGE kr1 TO sh2` sounds better, doesn't it?

Reply:
For manual execution, I think this is better for have this semantics: 

REDISTRIBUTE ... CHECK - dry run
REDISTRIBUTE ... APPLY - explicit request to start
REDISTRIBUTE ... - first check (abort if check failed), then start

Reply:
Can we do it in separate PR?